scripts/init.js

```diff
@@ -54,22 +62,20 @@ async function init() {
   initEagerImages();
 
   const blocks = [
-    document.body.querySelector(config.semanticBlocks[0]), 
-    ...document.querySelectorAll('[class]:not([class^=style]'), 
+    document.body.querySelector(config.semanticBlocks[0]),
+    ...document.querySelectorAll('[class]:not([class^=style]'),
     document.body.querySelector(config.semanticBlocks.slice(1).join(',')),
   ];
 
   const data = getInfos(blocks);
   const lcp = getLcp().map(({ name }) => includesInfo(data, name) || { name });
   const delay = window.raqnLCPDelay || [];
   const lazy = data.filter(
-    ({ name }) => !includesInfo(lcp, name) && !includesInfo(delay, name)
+    ({ name }) => !includesInfo(lcp, name) && !includesInfo(delay, name),
```

Comment:
Is the comma at the needed?

Reply:
We will use trailing all as linting rule in most projects due to:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Currently is missing here the eslint and prettier configs